src/layman/layer/filesystem/tasks.py

```diff
@@ -114,7 +114,11 @@ def finish_gdal_process(process):
 
         source_file = f'{layername}.tif' if normalize_filenames else input_path
         normalize_file_path = gdal.get_normalized_raster_layer_main_filepath(workspace, layername, source_file=source_file, )
-        process = gdal.compress_raster_file_async(output_file=normalize_file_path, file_to_compress=tmp_vrt_file, )
+        color_interpretations = gdal.get_color_interpretations(vrt_file_path or input_path)
+        band_to_create_mask_from = None if color_interpretations != ['Gray', 'Alpha'] or nodata_value is not None else 2
```

Comment:
Move this `if` into compress_and_mask_raster_file_async

Reply:
Comment this if, so we know the difference why GA needs a mask, but RGBA does not.